_init.py

```diff
@@ -31,7 +31,7 @@
 def _get_version_number_changelog():
     try:
         with open(os.path.join(ROOT_DIR, "CHANGELOG.md")) as f:
-            return re.search(r"(v[0-9]+\.[0-9]+\.[0-9]+)", f.readline()).group(0)
+            return f.readline()[4:-1]
```

Comment:
Probably better to just add to whatever the previous regex was not covering

Reply:
@thomasmckay I'm fine with this. Just need to make sure the name is on the first line.